h2o-extensions/target-encoder/src/test/java/ai/h2o/targetencoding/TargetEncoderMojoIntegrationTest.java

```diff
@@ -36,6 +35,7 @@
 import static ai.h2o.targetencoding.TargetEncoderHelper.addKFoldColumn;
 import static hex.genmodel.algos.targetencoder.TargetEncoderMojoModel.computeBlendedEncoding;
 import static hex.genmodel.algos.targetencoder.TargetEncoderMojoModel.computeLambda;
+import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 @RunWith(JUnitQuickcheck.class)
```

Comment:
This class was already using a runner:
```@RunWith(JUnitQuickcheck.class)```

So it would require some work to get some of the tests working.
What I could do probably, is extracting the tests that don't need quickcheck and move them to a different test class.